deploy/kubernetes/overlays/stable/kustomization.yaml

```diff
@@ -1,19 +1,4 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-bases:
-- ../../base
-images:
-- name: gke.gcr.io/gcp-compute-persistent-disk-csi-driver
-  # Don't change stable image without changing pdImagePlaceholder in
-  # test/k8s-integration/main.go
-  newName: gke.gcr.io/gcp-compute-persistent-disk-csi-driver
-  newTag: "v0.5.1-gke.0"
-- name: gke.gcr.io/csi-provisioner
-  newName: gke.gcr.io/csi-provisioner
-  newTag: "v1.2.1-gke.0"
-- name: gke.gcr.io/csi-attacher
-  newName: gke.gcr.io/csi-attacher
-  newTag: "v1.2.0-gke.0"
-- name: gke.gcr.io/csi-node-driver-registrar
-  newName: gke.gcr.io/csi-node-driver-registrar
-  newTag: "v1.1.0-gke.0"
+resources:
```

Comment:
I'm curious in the stable overlay why not set the image in the top level kustomization.yaml as what you did in prow-gke-release-staging-rc overlay and prow-gke-release-staging-head overlay.

Reply:
Because there will be layers built on top of the stable/controller, stable/node, and stable/rbac overlays that depend on images being set